src/UI/class-recommended-widget.php

```diff
@@ -27,6 +39,23 @@ final class Recommended_Widget extends WP_Widget {
 	 */
 	private $parsely;
 
+	/**
+	 * Default values of widget settings
+	 *
+	 * @var WidgetSettings
+	 */
+	private static $default_widget_settings = array(
+		'title'               => '',
+		'return_limit'        => 5,
+		'display_direction'   => 'vertical',
+		'published_within'    => 0,
+		'sort'                => 'score',
+		'boost'               => 'views',
+		'personalize_results' => false,
+		'img_src'             => 'parsely_thumb',
+		'display_author'      => false,
+	);
+
```

Comment:
In the code below we were using these as defaults if not defined so created a separate array for this and a merge function (same like parsely options)